kadai1/taxin/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/taxintt/gopherdojo-studyroom/kadai1/taxin/converter"
+)
+
+var (
+	imgFormat          string
+	convertedImgFormat string
+	dirPath            string
+	fileFormatList     []string
+)
+
+// Pass values that is passed when we execute cli tool to variables
+func init() {
+	flag.StringVar(&imgFormat, "f", "jpg", "画像ファイルの変更前のフォーマット(jpg/jpeg/png/gif)")
+	flag.StringVar(&convertedImgFormat, "o", "png", "画像ファイルの変更後のフォーマット(jpg/jpeg/png/gif)")
+	flag.StringVar(&dirPath, "d", ".", "画像が配置されているディレクトリのパス")
+	fileFormatList = []string{"jpg", "jpeg", "png", "gif"}
+	flag.Parse()
+}
+
+func main() {
+	if err := validateArgs(); err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		return
+	}
+	eachImgDirData := converter.ImgDirData{DirPath: dirPath, ImgFormat: imgFormat, ConvertedImgFormat: convertedImgFormat}
+	converter.WalkAndConvertImgFiles(eachImgDirData)
+}
+
+// validate arguments that are passed by.
+//
+// if you specify invalid file types or don't pass directory path that contains image files, it will raise an error.
+func validateArgs() error {
+	files, err := ioutil.ReadDir(dirPath)
+	if err != nil {
+		return err
+	}
+	if existsDir(files) {
```

Comment:
```suggestion
	if !existsDir(files) {
```

ディレクトリが存在していなければエラーを返すので`!`が必要だと思います